client/src/component/Skeleton/NavbarComponent/AppBarCollapse.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { MenuItem } from "@material-ui/core";
+import ButtonAppBarCollapse from "./ButtonAppBarCollapse";
+import Avatar from "@material-ui/core/Avatar";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+import Box from '@material-ui/core/Box';
+import Link from '@material-ui/core/Link';
+
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    position: "absolute",
+    right: 0,
+  },
+  buttonBar: {
+    [theme.breakpoints.down("xs")]: {
+      display: "none",
+    },
+  },
+  main_menu: {
+    backgroundColor: "white",
+    "& a .active": {
+      color: "orange",
+    },
+    "& a": {
+      textDecoration: "none",
+      fontWeight: "bold",
+      color: "black",
+      fontSize: "12px",
+      marginTop: "15px",
+      padding: "0px 10px 0px",
+    },
+    "& a:hover": {
+      color: "orange",
+    },
+    "& .avatar": {
+      marginLeft: "60px",
+    },
+  },
+}));
+
+const AppBarCollapse = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <ButtonAppBarCollapse>
+        <MenuItem >Home</MenuItem>
+        <MenuItem>Integration</MenuItem>
+        <MenuItem>Update Account</MenuItem>
+        <MenuItem> <Avatar
+            className={classes.main_menu.avatar}
+            alt="user 1"
+            src="/images/user.png"
+          /> John Doe</MenuItem>
+      </ButtonAppBarCollapse>
+      <Grid className={classes.buttonBar} id="appbar-collapse">
+        <Grid container className={classes.main_menu} >
+        
+          <Box mr={20} mt={1}>
+          <Link href="home">Home</Link>
+          <Link href="intergration">integration</Link>
+          <Link href="upgrade" className={classes.main_menu.active}>
+            Upgrade account
+          </Link>
+
+          </Box>
+         <Avatar
+            className={classes.main_menu.avatar}
+            alt="user 1"
+            src="/images/user.png"
+          />
+         
+          <a href="home">John Doe</a>
+          
+        </Grid>
+      </Grid>
+    </div>
+  );
+};
+
+export default AppBarCollapse;
```

Comment:
Is this the same as client/src/Component/NavbarComponent/AppBarCollapse.js ? 
We only want one component with the same name. If it's the same component, you can have just one and import it everywhere you need it; if not, edit the name so we can tell them apart.